domains/naresh.json

```
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "nareshkarthigeyan",
+        "email": "nareshkarthigeyan.2005@gmail.com"
+    },
+    "records": {
+        "CNAME": "cname.vercel-dns.com",
+        "A": "216.198.79.1"
```

again, no need to add this A records
```suggestion
        "CNAME": "cname.vercel-dns.com"
```